blockidentifier.cpp

```diff
@@ -264,6 +265,7 @@ void BlockIdentifier::parseDefinition(JSONObject *b, BlockInfo *parent,
                          name,
                          QMessageBox::Cancel, QMessageBox::Cancel);
   }
+#endif
```

Comment:
I think that rather than making this debug-only, the error message should be a bit more descriptive, while keeping it in the release version as well (so that it checks other authors' definition files as well).

How about `Failed to add block %1 definition from file %2, because it is already present in the loaded definitions.` ? I know it's not exactly what's happening, but it's still of much more help than the old message.

Reply:
File name is not available at that code position.

As the old text was only for me, it is obviously not descriptive. I tried a rephrase...